src/compas_rrc/ghpython/components/CompasRrc_GetFrame/metadata.json

```diff
@@ -0,0 +1,21 @@
+
+{
+    "name": "Get Frame",
+    "nickname": "Get Frame",
+    "category": "COMPAS RRC",
+    "subcategory": "Utility instructions",
+    "description": "Request the current frame of the robot.",
+    "exposure": 2,
+
+    "ghpython": {
+        "isAdvancedMode": true,
+        "iconDisplay": 2,
+        "inputParameters": [],
+        "outputParameters": [
+            {
+                "name": "ins",
+                "description": "Instruction instance."
```

Comment:
That doesn't look right for `GetFrame`.

Reply:
Maybe it's a crappy wording, but it is in fact what it's returning. All the "instruction" components return an instruction that can be executed with the `ExecuteBatch` component, not the type/data implied in the name (i.e. a `Frame` in this case).

Reply:
oh ok